scripts/Diffraction/isis_powder/routines/common.py

```diff
@@ -79,24 +79,28 @@ def crop_banks_using_crop_list(bank_list, crop_values_list):
     :param crop_values_list: The cropping values to apply to each bank
     :return: A list of cropped workspaces
     """
-    if not isinstance(crop_values_list, list):
-        raise ValueError("The cropping values were not in a list type")
+    if not isinstance(crop_values_list, list) and not isinstance(crop_values_list, tuple):
```

Comment:
Python has a useful [`collections`](https://docs.python.org/2/library/collections.html#collections.Sequence) module where it defines abstract types so you can do thinks like
```python
isinstance(my_obj, collections.Sequence)
```
that will then check the object *looks* like a sequence without having to require it to be specifically a list/tuple. For example a numpy array should be allowable but would fail here. Can we use the `collections` module both here and on line 89?

Reply:
I'm not sure it could be used on line 89, as it checks for a tuple specifically to know if it should apply one cropping to all spectra. Unless there's a better way of doing that.

Reply:
I think we need to check on the sizes don't we? We definitely want to allow a single `[min, max]` too. Can we check if we have a sequence of sequences (just checking the first element is probably sufficient and then if the first element is not a sequence we assume we have a range?

Reply:
Okay, my understanding was that `[(0.5, 0.9), (0.2, 0.8)]` or `(0.05, 0.95)` would be the formats used, but your suggestion makes more sense.